docs/content/en/docs/reference/support/support-versions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Version support"
+linkTitle: "Version support"
+weight: 50
+description: >
+  EKS Anywhere and Kubernetes version support policy
+---
+
+To see supported versions of Kubernetes for each release of EKS Anywhere, and information about that support, refer to the content below.
+
+
+## Kubernetes support
+
+Each EKS Anywhere version generally includes support for multiple Kubernetes versions, with the exception of the initial few releases.
+Starting from EKS Anywhere version 0.11, the latest version supports at least four recent versions of Kubernetes.
+The end of support date of a Kubernetes version aligns with Amazon EKS in AWS as documented on the [Amazon EKS Kubernetes release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar). 
+
+Common vulnerabilities and exposures (CVE) patches and bug fixes, including those for the supported Kubernetes versions, are back-ported to the latest EKS Anywhere version (version n) and the previous one version (version n-1).
+The following table shows EKS Anywhere version support for different Kubernetes versions:
+
```

Comment:
this list will need updating soon

Reply:
Yes, this is meant to be an evolving doc page.